### DISCUSSION TITLE
ETA for 4.0?

### DISCUSSION THREAD

**Discussion by kennedymeadows** [netbox-community/netbox#15469] [General]
I'm currently doing a big initiative for our company's networking team which Netbox is a central pillar of. I've had a PoC running for a few months now and am starting to look at a more robust deployment. I was wondering if there's an ETA for the 4.0 release? I think it will make more sense to focus my efforts once this release is available.

**Comment by jeremystretch** [comment]
We do minor (or in this case, major) releases every four months. Look for the first v4.0 beta in 2-3 weeks, with the final release due the week of April 29th.

**Reply by kennedymeadows** [reply]
Thanks Jeremy, looking forward to getting my hands on it.
